packages/flutter_tools/test/integration.shard/migrate_config_test.dart

```diff
@@ -0,0 +1,192 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
+
+import 'package:file/file.dart';
+import 'package:flutter_tools/src/migrate/migrate_config.dart';
+
+import '../src/common.dart';
+import '../src/context.dart';
+import 'test_data/migrate_project.dart';
+import 'test_driver.dart';
+import 'test_utils.dart';
+
+
+void main() {
+  Directory tempDir;
+  FlutterRunTestDriver flutter;
+
+  setUp(() async {
+    tempDir = createResolvedTempDirectorySync('run_test.');
+    flutter = FlutterRunTestDriver(tempDir);
+  });
+
+  tearDown(() async {
+    await flutter.stop();
+    tryToDelete(tempDir);
+  });
+
+  testUsingContext('parse simple config file', () async {
+    // Flutter Stable 1.22.6 hash: 9b2d32b605630f28625709ebd9d78ab3016b2bf6
+    final MigrateProject project = MigrateProject('version:1.22.6_stable');
+    await project.setUpIn(tempDir);
+
+    final File configFile = tempDir.childFile('.migrate_config');
+    configFile.createSync(recursive: true);
+    configFile.writeAsStringSync('''
+# Generated section.
+platform: 'root'
+createRevision: 'abcdefg1234567'
+baseRevision: 'abcdefg1234567base'
+
+# User provided section
+
+# List of Local paths (relative to this file) that should be
+# ignored by the migrate tool.
+#
+# Files that are not part of the templates will be ignored by default.
+unmanagedFiles:
+  - 'lib/main.dart'
```

Comment:
should this support the glob pattern?  for example: `lib/**/*.dart` seems reasonable to me.

Reply:
Is there a lib we typically use to support this? It seems we support just simple directories and file URIs in pubspec.yaml

Reply:
Yup: https://github.com/dart-lang/glob